src/components/icon/icon.stories.mdx

```diff
@@ -36,13 +45,62 @@ For a list of available icons, visit [the Icons page](/docs/design-icons--page).
       name: 'heart',
       inline: false,
     }}
-    argTypes={{
-      name: { type: { name: 'string' } },
-      fallback: { type: { name: 'string' } },
-      inline: { type: { name: 'boolean' } },
-      class: { type: { name: 'string' } },
-      title: { type: { name: 'string' } },
+    argTypes={defaultArgs}
+  >
+    {(args) => iconStory(args)}
+  </Story>
+</Canvas>
+
+## Muted
+
+By setting the `muted` property to `true`, the icon will become slightly opaque.
+
+<Canvas>
+  <Story
+    name="Muted"
+    args={{
+      name: 'heart',
+      inline: false,
+      muted: true,
+    }}
+    argTypes={defaultArgs}
+  >
+    {(args) => iconStory(args)}
+  </Story>
+</Canvas>
+
+## Sizes
+
+By setting the `size` property to the [name of a `size.$icon` token](/docs/design-tokens-size--page), such as `large`, the icon will be scaled to the appropriate size.
```

Comment:
@spaceninja Can you also add this new `size` property to the "**Template Properties**" list below? Thanks! 👍🏽